mobile/src/main/java/org/openhab/habdroid/util/PrefExtensions.kt

```diff
@@ -120,3 +121,8 @@ fun SharedPreferences.Editor.updateDefaultSitemap(sitemap: Sitemap?) {
         putString(Constants.PREFERENCE_SITEMAP_LABEL, sitemap.label)
     }
 }
+
+fun SharedPreferences.Editor.disableItemUpdatePref(context: Context, key: String) {
+    val item = context.getPrefs().getString(key).toItemUpdatePrefValue().second
+    putString(key, "false|$item")
+}
```

Comment:
The definition extension function seems a little unclear/misleading to me. You call it on a specific prefs editor, but fetch the used value from `getPrefs()`, which the editor may or may not belong to. I think it's better to not make this an extension function and inline this code into the (one) call place.